app/articles/server/methods/user.js

```diff
@@ -0,0 +1,88 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { API } from '../utils/url';
+import { settings } from '../../../settings';
+
+const api = new API();
+
+function addUser(user, accessToken) {
+	const data = {
+		user: [{
+			rc_username: user.username,
+			role: 'Author', // User can add itself as Author, even if he/she is admin in RC
+			rc_uid: user._id,
+			rc_token: accessToken,
+		}],
+	};
+	return HTTP.call('POST', api.createAccount(), { data, headers: { 'Content-Type': 'application/json' } });
+}
+
+function userExist(user, accessToken) {
+	const data = {
+		user: [{
+			rc_uid: user._id,
+			rc_token: accessToken,
+		}],
+	};
+	const response = HTTP.call('GET', api.userExist(), { data, headers: { 'Content-Type': 'application/json' } });
+	return response.data && response.data.users[0] && response.data.users[0].exist;
+}
+
+function inviteSetting() {
+	const response = HTTP.call('GET', api.invite());
+	const { settings } = response.data;
+
+	if (settings && settings[0] && settings[0].key === 'invite_only') {
+		return settings[0].value;
+	}
+	// default value in Ghost
+	return false;
+}
+
+function redirectGhost() {
+	return {
+		link: api.siteUrl(),
+		message: 'Ghost is Set up. Redirecting.',
+	};
+}
+
+Meteor.methods({
+	redirectUserToArticles(accessToken) {
```

Comment:
This function to be refactored to make it smaller. Reduce the if-else nesting levels.